### PR TITLE
Allow user to change delay between scans

### PR DESCRIPTION
**Pull request by nestoralonsovina**
Add input to allow the user to change the delay between scans

Related #183

### CODE REVIEW

**Comment by odahcam** [comment]
Thank you very much for your contribution!

File: projects/zxing-scanner/src/lib/zxing-scanner.component.ts [hunk 1]
```diff
@@ -81,6 +81,12 @@ export class ZXingScannerComponent implements AfterViewInit, OnDestroy {
   @Input()
   autofocusEnabled: boolean;
 
+  /**
+   * Modify delay between scans (default is 500ms)
+   */
+  @Input()
+  timeBetweenScansMillis = 500;
```
Comment (odahcam) [inline projects/zxing-scanner/src/lib/zxing-scanner.component.ts:88]: I would just like to call it `timeBetweenScans` without the `Millis`.
```suggestion
  timeBetweenScans = 500;
```

File: projects/zxing-scanner/src/lib/zxing-scanner.component.ts [hunk 2]
```diff
@@ -674,7 +680,7 @@ export class ZXingScannerComponent implements AfterViewInit, OnDestroy {
   private getCodeReader(): BrowserMultiFormatContinuousReader {
 
     if (!this._codeReader) {
-      this._codeReader = new BrowserMultiFormatContinuousReader(this.hints);
+      this._codeReader = new BrowserMultiFormatContinuousReader(this.hints, this.timeBetweenScansMillis);
```
Comment (odahcam) [inline projects/zxing-scanner/src/lib/zxing-scanner.component.ts:683]:
```suggestion
      this._codeReader = new BrowserMultiFormatContinuousReader(this.hints, this.timeBetweenScans);
```

**Comment by krumholz** [comment]
I just updated to v3.0.1. I have scanComplete and scanFailure handlers and immediately after enabling the scanner they are still firing hundreds of times a second. This is the same as before I updated to the latest version.

It appears that I'm either not using this feature correctly, I'm not understanding what it does, or it's not working. I've tried variations of: `<zxing-scanner timeBetweenScans="1000"></zxing-scanner>`.

Would someone be able to add details about how to use this feature and/or verify that it's working as expected?

**Pull request merged**